<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileUser</class>
 <widget class="QMainWindow" name="ProfileUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pB_StartLevel2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_StartLevel1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_StartLevel3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_closeProfile">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
